server/app/jobs/deployment_cleanup_job.rb

```diff
@@ -21,7 +21,7 @@ def perform
 
   def destroy_old_deployments
     GridService.each do |s|
-      return if s.grid_service_deploys.finished.count <= 100
+      next if s.grid_service_deploys.finished.count <= 100
```

Comment:
This should work correctly, but the same code could be written more clearly without the use of `return`/`next`:

```ruby
def destroy_old_deployments
  GridService.each do |s|
    cleanup_old_deployments(s) if s.grid_service_deploys.finished.count > 100
  end
end

def cleanup_old_deployments(grid_service)
  info "cleaning old deployments for service #{service.to_path}"
  ...
end
```